code/lib/codemod/src/index.ts

```diff
@@ -47,14 +47,19 @@ export async function runCodemod(codemod: any, { glob, logger, dryRun, rename, p
   // jscodeshift/prettier know how to handle .ts/.tsx extensions,
   // so if the user uses one of those globs, we can auto-infer
   let inferredParser = parser;
+
   if (!parser) {
     const extension = path.extname(glob).slice(1);
     const knownParser = jscodeshiftToPrettierParser(extension);
     if (knownParser !== 'babel') inferredParser = extension;
   }
 
   const files = await globby([glob, '!**/node_modules', '!**/dist']);
+  const extensions = new Set(files.map((file) => path.extname(file).slice(1)));
```

Comment:
@valentinpalkovic Did you try this on `.stories.mdx` files?

Reply:
No, I haven't I can put it on my todo list